scripts/ufm_health/README.md

```diff
@@ -0,0 +1,35 @@
+UFM Health
+--------------------------------------------------------
+
+
+This script is used to run UFM Health Report via the UFM APIs.
```

Comment:
please add a brief description and link to :
https://docs.nvidia.com/networking/display/UFMEnterpriseUMv69/UFM+Health+Tab